src/css/components/Nav.scss

```diff
@@ -40,40 +40,83 @@
 
   .Nav-sectionHeading {
     padding: 8px;
+    border-bottom: 1px solid color-neutral(4);
   }
 
   .Nav-topLevelHeading {
     padding: 6px 8px;
     border-radius: 5px;
+    border-bottom: 1px solid color-neutral(4);
 
     &:hover {
       background-color: color-neutral(3);
     }
   }
 
-  .Nav-node > .Nav-node {
-    padding-left: 16px;
-  }
+  .Nav-node {
+    padding: 0.75rem 1rem 0.75rem 1.75rem;
```

Comment:
Is Drew cool with the shift from our grid scaling to something more flexible like REM?

Reply:
Chatted with Drew to get the spacing correctly on Nav items.